src/components/organisms/Room/Room.tsx

```diff
@@ -197,14 +197,16 @@ const Room: React.FC<RoomProps> = ({
   // Only allow the first bartender to appear as bartender
   const userIdentity = room?.localParticipant?.identity;
   const meIsBartender =
-    users && userIdentity
-      ? users[userIdentity]?.data?.[roomName]?.bartender
+    worldUsersById && userIdentity
+      ? worldUsersById[userIdentity]?.data?.[roomName]?.bartender
       : undefined;
```

Comment:
```suggestion
    worldUsersById?.[userIdentity]?.data?.[roomName]?.bartender ?? false;
```

Reply:
Couldn't apply the whole suggested change, but did change the expression slightly

Reply:
What was the issue in applying the change?

Reply:
Guessing this is how it was changed to? https://github.com/sparkletown/sparkle/pull/1047/files?file-filters%5B%5D=.tsx#diff-ab9b758a67c14d5beecc2d0ae0320ea4201dd3ce3f2769967bf8bdc16fd4f34cR200-R203

The ternary is redundant as that's what the safe accessor methods (`?`) are for, when it is `undefined`, the whole expression returns `undefined`.

I think the only thing that may not have been able to be directly applied is the `?? false` part.